.github/workflows/test.yaml

```
@@ -6,20 +6,20 @@ on:
       - ros2
 
 jobs:
-  build_and_test_source_rolling:
+  build_and_test_source_kilted:
     runs-on: ubuntu-latest
     container:
       image: rostooling/setup-ros-docker:ubuntu-noble-latest
```

It has to use the testing apt repo for Kilted since it hasn't been released. Since this uses a Docker image from setup-ros-docker, it could use the new `setup-ros-docker-ubuntu-noble-testing` image which uses the testing repo:

```suggestion
      image: ghcr.io/ros-tooling/setup-ros-docker/setup-ros-docker-ubuntu-noble-testing
```

Same for the other job